python/GafferArnoldTest/IECoreArnoldPreviewTest/RendererTest.py

```diff
@@ -589,8 +604,19 @@ def testExrMetadata( self ) :
 
 			arnold.AiASSLoad( self.temporaryDirectory() + "/test.ass" )
 
+			# test if data was added correctly to existing data
 			exrDriver = arnold.AiNodeLookUpByName( "ieCoreArnold:display:exrTest" )
 			customAttributes = arnold.AiNodeGetArray( exrDriver, "custom_attributes" )
+			customAttributesValues = set([ arnold.AiArrayGetStr( customAttributes, i ) for i in range( 2 ) ])
```

Comment:
Can we use the array length rather than hardcode 2, to be sure we don't accidentally put non-header parameters into the header?